students/aravichander/lesson06/Mailroom3.py

```diff
@@ -1,19 +1,21 @@
-#Mailroom #2
+#Mailroom #3&4
 
 import sys
 import json
 import unittest 
 
-Donors = {"Bill Gates":[10000,12000],
-		  "Jeff Bezos":[50],`
-		  "Mark Zuckerberg": [500,600,700],
-		  "Paul Allen": [250,350,450],
-		  "King of Siam": [200,250]
-		  }
+Donors = {}
+def createlist():
```

Comment:
use names that describe what the function does:

this is crating a dict, not a list :-) -- but also -- what kind of list(dict?) -- somethign like:

`create_sample_donors()`

would be more clear.